src/test/resources/mcmod.info

```diff
@@ -1,16 +1,8 @@
 [
 {
-  "modid": "config_test",
```

Comment:
This breaks the other test, you must define both mods here.

Reply:
I didn't realize that "mcmod"s are Json files, too, hah. I learned it now, thanks.